### PR TITLE
hooks: pre-commit hook to duplicate file

### PR DESCRIPTION
**Pull request by omertuc**
We have a file that's always a duplicate of another file, until we can get rid of this requirement a pre-commit hook to take care of it would be nice

### CODE REVIEW

File: training/nvidia-bootc/duplicated/ilab-wrapper/ilab [hunk 1]
```diff
@@ -60,23 +60,30 @@ fi
...
 # reasonable way for the current user to access the root's user container
 # storage.
-CURRENT_USER_NAME=$(id --user --name)
-CURRENT_USER_SUBUID_RANGE=$(awk \
-    --field-separator ':' \
-    --assign current_user="$CURRENT_USER_NAME" \
-    --assign current_uid="$UID" \
-    '$1 == current_user || $1 == current_uid {print $2 ":" $3}' \
-    /etc/subuid)
+if [[ "$UID" == 0 ]]; then
```
Comment (rhatdan) [inline training/nvidia-bootc/duplicated/ilab-wrapper/ilab]: This should not be part of this PR.

Comment (omertuc) [reply training/nvidia-bootc/duplicated/ilab-wrapper/ilab]: I disagree, it's syncing the duplicated wrapper as expected from the hook

Comment (omertuc) [reply training/nvidia-bootc/duplicated/ilab-wrapper/ilab]: But looks like #731 already did that

**Review by rhatdan — APPROVED** [review]
LGTM
